src/insns/load_tag_perms.adoc

```diff
@@ -0,0 +1,11 @@
+Resulting value of `cd`::
+The tag value written to `cd` is 0 if the tag of the memory location loaded is
+0 or the authorizing capability (<<ddc>> or `cs1`) does not grant <<c_perm>>.
```

Comment:
This text is repeated in some of the instruction listings (e.g. LC)

Reply:
Do you mean I should factor this out?

Reply:
Ah I see what you meant. Fixed now.